core/core-configuration-layer.el

```diff
@@ -796,10 +804,21 @@ If called with a prefix argument ALWAYS-UPDATE, assume yes to update."
          (upgrade-count (length update-packages))
          (upgraded-count 0)
          (update-packages-alist))
+    (when configuration-layer--skipped-packages
+      (spacemacs-buffer/append
+       (format "--> Warning: skipping %s package(s): %s\n"
```

Comment:
@bmag @syl20bnr How about this for the message? 

"--> Warning: cannot update %s package(s), possibly due to a temporary network problem: %s\n"